src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/managed/ManagedStrategy.kt

```diff
@@ -160,9 +152,10 @@ abstract class ManagedStrategy(
     protected abstract fun chooseThread(iThread: Int): Int
 
     /**
-     * Returns all data to the initial state.
+     * Resets all internal data to the initial state and initializes current invocation to be run.
      */
-    protected open fun initializeInvocation() {
+    override fun initializeInvocation() {
+        super.initializeInvocation()
```

Comment:
Why do you need this call?

Reply:
Added it just for safety. 
In case if in the future we add some code to super method (which is currently empty, yes).